src/actionSheetIOS.re

```diff
@@ -1,6 +1,8 @@
 type actionSheetConfig;
 
 type shareActionSheetConfig;
+external shareActionSheetConfig: Js.t('a) => shareActionSheetConfig =
+  "%identity";
```

Comment:
`%` is added intentionally?

Reply:
Probably, see 
https://bucklescript.github.io/docs/en/intro-to-external#special-identity-external

Reply:
Nice, I didn't know about it. Thanks for the explanation ;)